cards/ecg-right-posterior-leads/card.md

```diff
@@ -52,3 +52,6 @@ Hints of an associated posterior infarct:
 2.  Upright T wave waves (in later stage)
 
 ## References
+
+1. [Edhouse J, Brady WJ, Morris F. ABC of clinical electrocardiography: Acute myocardial infarction-Part II. BMJ. 2002; 324: 963-6.] (https://doi.org/10.1136/bmj.324.7343.963)
```

Comment:
Yeah, down in the dirt with remark now. I love the configurability for your own use case.

That convention of a tab stop is a little tricky to explain to non-coders. Lists in *Paucis Verbis* cards probably shouldn't be going longer than 10 items usually anyway. I think a single space is better.

Will it be hard to revert all the lists that got changed by your beautify process?